vllm/model_executor/layers/rejection_sampler.py

```diff
@@ -368,7 +368,7 @@ def _smallest_positive_value(self) -> float:
 # Note that we always sample with replacement.
 # probs will be modified in place, but this is fine, as we pass
 # in a copy already.
-@torch.jit.script
+@torch.compile
```

Comment:
QQ: Should we add `dynamic=True` to preserve the behavior of `torch.jit.script`?

Reply:
IIUC, otherwise `torch.compile` triggers re-compilation after the very first run. Not sure whether this violates anything, but I feel like `dynamic=True` doesn't hurt anything?

Reply:
good point, added in [c0f6094](https://github.com/vllm-project/vllm/pull/10406/commits/c0f60948f3d4452ca9f35443d2e1082d9babaa51)